docs/charter.md

```diff
@@ -108,12 +111,15 @@ The _CPAN-SEC_ concerns itself with a limited domain:
 
 The _CPAN-SEC_ has **no power, agency or special privileges over CPAN distributions or their authors**, except for those releases done by the team itself.
 
-The working group publishes information, advisories and recommendations, but cannot force anyone to follow them. Our goal is to facilitate in-good-faith productive and security-focused discussions, with the intention to assist in resolving any issues in a responsible and timely manner.
+The working group publishes information, advisories and recommendations, but cannot force anyone to follow them.
+Our goal is to facilitate in-good-faith productive and security-focused discussions, with the intention to assist in resolving any issues in a responsible and timely manner.
 
```

Comment:
@timlegge commented in the cryptpad:

> *COMMENT**: timlegge - this is concerning.  While I realize the history of FIRSTCOME etc. having full rights to a module/namespace, leaving serious vulnerabilities in place is an issue.  There should be some thought given to how, when and why a module might be flagged, removed, fixed, or its FIRSTCOME or MAINT reassigned.  If there are no approved processes in place, it will be difficult to deal with when a serious issue is not being resolved.  Unresolved Security issues are a community issue and affect the reputation of the community as a whole.  There must be a way to deal with them.

Reply:
@timlegge & @garu, after our chat on IRC, is there a wording that you think we would be happy with here?

Reply:
@sjn @garu Based on the community norms and status quo there is not much to do there I guess.

Instead make some recommendations to potentially improve security response like trying to get metacpan to include a vulnerable flag that could be toggled (not sure by whom)

The VULNERABLE notice on metacpan might push the issue more than anything else.

Reply:
ok, though is this behavior something we should mention in the charter?

Reply:
The info as listed would likely cover that I think under recommendations.